### DISCUSSION TITLE
The Immich core team goes full-time

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#9206] [Announcements]
**Immich is joining [FUTO](https://futo.org/)!**

Since the beginning of this adventure, my goal has always been to create a better world for my children. Memories are priceless, and privacy should not be a luxury. However, building quality open source has its challenges. Over the past two years, it has taken significant dedication, time, and effort.

Recently, a company in Austin, Texas, called FUTO contacted the team. FUTO strives to develop quality and sustainable open software. They build software alternatives that focus on giving control to users. From their mission statement:

“Computers should belong to you, the people. We develop and fund technology to give them back.”

FUTO loved Immich and wanted to see if we’d consider working with them to take the project to the next level. In short, FUTO offered to:

- Pay the core team to work on Immich full-time
- Let us keep full autonomy about the project’s direction and leadership
- Continue to license Immich under AGPL
- Keep Immich’s development direction with no paywalled features
- Keep Immich “built for the people” (no ads, data mining/selling, or alternative motives)
- Provide us with financial, technical, legal, and administrative support

After careful deliberation, the team decided that FUTO’s vision closely aligns with our own: to build a better future by providing a polished, performant, and privacy-preserving open-source software solution for photo and video management delivered in a sustainable way.

Immich’s future has never looked brighter, and we look forward to realizing our vision for Immich as part of FUTO.

See our post [here](https://immich.app/blog/2024/immich-core-team-goes-fulltime) for full details about this change, including answers to frequently asked questions. If you have more questions, we’ll host a Q&A live stream on May 9th at 3PM UTC (10AM CST). [You can ask questions here](https://www.live-ask.com/event/01HWP2SB99A1K8EXFBDKZ5Z9CF), and the stream will be live [here on our YouTube channel](https://youtube.com/live/cwz2iZwYpgg).

Cheers,

The Immich Team

**Comment by diyoyo** [comment]
Congrats!!
now that I think about it, I'd love to read your retrospective story on this journey, like the ups and downs, the struggles, the happy encounters, the financial quests, etc., like in a personal tone.
I guess we all have in mind softwares that we'd like to "adopt and adapt" in a floss way, but there is always the little voice saying "don't waste your time on something that already fits the needs of many". So your story could be an encouraging counter-voice.

**Reply by danieldietzler** [reply]
That's a question you could ask in the livestream and if you're lucky they'll talk about it! :)

**Comment by RobustMarker** [comment]
Thank you for what you have done so far, even in this short period Immich is my main method to store photos (yes I am doing 3:2:1 backups). I hope Immich's team keep the same values and views, as you said yourself "privacy should not be a luxury". I really hope this project continues to improve!

**Comment by Meliox** [comment]
Wow. Congrats. I have just been part of the journey for 1.5 years, yet I have still seen how the community has developed and how dedicated you all have been.
FUTO vision is really awesome and I am very happy to see that it aligns with immich and that you can continue to do what makes immich fantastic. This will be a big change for both immich and the team. Wish you the best for the next step of the journey.

**Comment by Cp0204** [comment]
Thanks to Imrich team for everything, hope to join FUTO, keep the initial heart, get better development. ❤️

**Comment by CadeusTheGreat** [comment]
Huge congratulations to the Immich team! Exciting news & can't wait to see where development goes from here! :D

**Comment by cvwright** [comment]
Welcome aboard!

**Comment by petabyt** [comment]
Welcome and congratulations!

**Comment by A-wels** [comment]
Happy that you get to work on this project full time! <3 This project is awesome.

**Comment by Ahwxorg** [comment]
Thank you for your work so far! Hoping to be there for Immich's further improvement!

**Comment by jkaberg** [comment]
I think this is great news and much appreciated as long as "nothing" changes license wise, and I truly hope so. However experiences in the past has shown otherwise but I'll give it the benefit of the doubt! 🤞 ❤️

**Comment by pacnwsoundstream** [comment]
I really appreciate immich and all you and the team have done. I am also a fan of FUTO and their mission. Not usually do these situations leave me excited or optimistic. In this case I am eager to see what the future has in store for immich!

**Comment by Wraient** [comment]
Just set up my homelab server with Immich yesterday and today there is a huge announcement about how it'll get better. Couldn't ask for a better timing.

**Comment by kikiThinking** [comment]
Great! Looking forward to more changes in Immich, looking forward to Immich becoming even better.

**Comment by SandiyosDev** [comment]
F*ing love it, I started using Immich a while back and have always kept track of new pulls and issues, I can't code well, but I'd try and help other devs in solving issues.
I love every line (of code) of Immich, and congratulations Alex, this is certainly an incredible ride.

**Comment by Zakmaf** [comment]
As someone who's never tipped you guys, even though I use and support your project, all I can say is : Good for you!
I wish you all the success your deserve.
I have no worries about immich, since it's an open source project, the real judgement comes from the usage and users, not from philosophy and politics before hand.

**Comment by Leenus533** [comment]
Well deserved 😆.  Absolutely love this repo.

**Comment by principino1984** [comment]
I do congrats with you guys for this awesome project that finally gave me the hope to be free from google photos and iCloud. I really hope this project will stay forever with this "license" and till it will be a real opensource program I will keep beeing an active supporter.
Thanks

**Comment by Wildcarde** [comment]
While I'm sure the funding from this is great.  Joining a group actively platforming Curtis Yarvin seems less great (https://www.britannica.com/biography/Curtis-Yarvin). Like the guy doesn't hide being absolutely awful and even a generous read seems to indicate the founder of FUTO is buddies with him and likes having him on streams: https://drewdevault.com/2025/10/22/2025-10-22-Whats-up-with-FUTO.html

I'm not sure how that squares with your own code of conduct and the community you assert that you are trying to foster: https://github.com/immich-app/immich/blob/main/CODE_OF_CONDUCT.md

**Reply by Prolonz** [reply]
What's the suggestion here? That Immich should go back to being volunteer only? The team, who uprooted their lives to work on Immich full time, should give it up their Jobs? All because someone involved in the parent organization is friendly with a guy who says things you don't like. 

This is cancel culture rearing it's ugly head. I highly recommend you read and have a long think about The Coddling of the American Mind.

**Reply by ghost** [reply]
@Prolonz The suggestion here is that associating your project with a fascist-adjacent, washed-up millionaire’s scummy vanity company is not a good look for anyone. I have created a separate discussion for it here: https://github.com/immich-app/immich/discussions/23176

**Reply by ghost** [reply]
> This is cancel culture rearing it's ugly head. I highly recommend you read and have a long think about The Coddling of the American Mind.

@Prolonz, ok clear, so you support FUTO's politics. Just be open about it, instead of stating it in such a roundabout manner.

**Comment by Wildcarde** [comment]
I'm not sure there is a quick fix for this. Which is why I'm asking where
this gets explained and how it's compatible with immich's stated goals for
their community that existed long before joining with this new gaggle of
weirdos.

And let's be clear, Yarvin isn't 'some guy'. He's a dude so toxic YouTube
kicked him off, whose big brain idea is that everyone not in tech should be
serfs to tech bros.
